lint

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python
 import sys
 
-from tools.lint import lint
+try:
+    from tools.lint import lint
+except ImportError:
+    print("tools.lint not found.  Did you forget to run "
+          '"git submodule update --init --recursive"?')
+    sys.exit(2)
```

Comment:
I considered that, but preferred this way because the built-in Python error from an uncaught exception is lengthy and not helpful to the user.